packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -179,6 +179,14 @@ export function makeVatKeeper(
     }
   }
 
+  function markAsDead() {
+    storage.set(`${vatID}.dead`, true);
```

Comment:
I think the HostDB is string-key string-value (no automatic JSONification) so this needs to be `'true'`, not `true`